### PR TITLE
Added rotation of circular viewer.

### PR DESCRIPTION
**Pull request by digithed**
Addresses #27 

- Added scroll functionality to the circular viewer so that the viewer rotates around its origin on scroll

### CODE REVIEW

File: src/components/Ariadne/CircularViewer/CircularViewer.tsx [hunk 1]
```diff
@@ -28,6 +34,8 @@ export const CircularViewer = ({ sequence, size, annotations }: Props) => {
         className={`stroke-current`}
         width={sizeX}
         height={sizeY}
+        transform={`rotate(${scrollVal})`}
+        onWheel={(e) => handleScroll(e)}
```
Comment (ninjha01) [inline src/components/Ariadne/CircularViewer/CircularViewer.tsx:38]: Any reason we went with onWheel instead of something like onScroll?

**Review by ninjha01 — CHANGES_REQUESTED** [review]
Looks good, just some quick questions

File: src/components/Ariadne/CircularViewer/CircularViewer.tsx (hunk 1 above)
Comment (digithed) [reply src/components/Ariadne/CircularViewer/CircularViewer.tsx:38]: From stackoverflow: "onwheel : Occurs when the mouse wheel is rolled up or down over an element. onscroll Occurs when an element's scrollbar is being scrolled"

File: src/components/Ariadne/CircularViewer/CircularIndex/CircularIndex.tsx [hunk 2]
```diff
@@ -18,14 +18,16 @@ export const CircularIndex = ({
...
       <svg className={`bg-brand-800 fill-current text-brand-800`}>
-        <circle
-          cx={cx}
-          cy={cy}
-          r={radius * 0.75}
-          fill="none"
-          stroke="currentColor"
-          strokeWidth={2}
-        />
+        <g>
```
[the remark this answers is not in the document]
Comment (digithed) [reply src/components/Ariadne/CircularViewer/CircularIndex/CircularIndex.tsx:21]: Nope, remnants of some testing

**Review by ninjha01 — APPROVED** [review]

**Comment by ninjha01** [comment]
@digithed did a bit of cleanup and made sure the text in the middle of the viewer stays stationary. If it looks good to you LGTM!

**Comment by ninjha01** [comment]
I have something I want to work that I want this merged in for, so I'm going to go ahead and merge but we can open an issue if there's anything you want to change!

**Pull request merged**